app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -174,21 +174,22 @@ function insertArticles(articles) {
   }
 }
 
-function fetchNextPodcastPage(el){
+function fetchNextPodcastPage(el) {
   fetchNext(el, "/api/podcast_episodes", insertArticles)
 }
 
-function algoliaPaginate(tag){
-  var filters = [];
+function algoliaPaginate(tag, requiresApproval) {
+  var tagFilters = [];
   if (tag && tag.length > 0) {
-    filters.push(tag);
+    tagFilters.push(tag);
   }
 
   var searchObj = {
       hitsPerPage: 15,
       page: nextPage,
       attributesToHighlight: [],
-      tagFilters: filters,
+      tagFilters: tagFilters,
+      filters: (requiresApproval === 'true') ? 'approved:true' : ''
```

Comment:
Any reason `requiresApproval` is a string and not a boolean? Is it coming from a DOM attribute?

Reply:
> Any reason `requiresApproval` is a string and not a boolean? Is it coming from a DOM attribute?

Correct, It's coming from the DOM